tox.ini

```
@@ -98,16 +98,10 @@ commands =
 [docs]
 docs_dir = {tox_root}{/}docs
 build_dir = {[docs]docs_dir}{/}_build
-# replace when Sphinx>=7.3 and Python 3.8 is dropped:
-#  -T => --show-traceback
-#  -W => --fail-on-warning
-#  -b => --builder
-#  -v => --verbose
-#  -a => --write-all
-#  -E => --fresh-env
-sphinx_args = -T -W --keep-going --jobs auto
+sphinx_args = --show-traceback --fail-on-warning --keep-going --jobs auto
 
 [testenv:docs{,-lint,-all,-live}]
+basepython = python3.12
```

For consistency :) or perhaps the coverage section could be updated to match this

```suggestion
base_python = py312
```

---

WHY ARE THERE TWO VALID SYNTAXES FOR THIS!!>!!KJHKJH!OI&!*^*&!